snapshotter/processor_distributor.py

```diff
@@ -330,6 +334,23 @@ async def _load_projects_metadata(self):
                             data_source,
                         )
                     project_config.projects = list(project_data)
+                all_projects += [f'{project_type}:{project}:{settings.namespace}' for project in project_config.projects]
```

Comment:
I think this should be `[f'{project_type}:{project.lower()}:{settings.namespace}' for project in project_config.projects]` instead.

Have we tested this somewhere?

Reply:
Added the change here: c51bf01

Yes it was tested on my node. The projects were already lowercase in the config, but this is a good change to force consistency.

Reply:
looks good now!